### PR TITLE
Reword comments in Mutect2.wdl

### PR DESCRIPTION
**Pull request by takutosato**
In particular, do not call the old orientation bias filter deprecated

### CODE REVIEW

**Comment by takutosato** [comment]
@LeeTL1220 please review

File: scripts/mutect2_wdl/mutect2.wdl [hunk 1]
```diff
@@ -796,6 +796,7 @@ task CollectF1R2Counts {
     }
 }
 
+# Learning step of the orientation bias mixture model, which is the recommended orientation bias filter as of September 2017
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: 2017?  2018?

File: scripts/mutect2_wdl/mutect2.wdl [hunk 2]
```diff
@@ -737,6 +736,7 @@ task CollectSequencingArtifactMetrics {
     }
 }
 
+# Data collection step of the orientation bias mixture model, which is the recommended orientation bias filter as of September 2017
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/mutect2.wdl]: 2017?  2018?

**Comment by LeeTL1220** [comment]
@takutosato @droazen Can we make sure that this gets in the next release?  Users may misinterpret the deprecation warning that is removed in this PR.

**Comment by takutosato** [comment]
@LeeTL1220 thanks for your comments. Yes, I will make suggested changes by the end of the week, if not later today

File: scripts/mutect2_wdl/mutect2.wdl [hunk 3]
```diff
@@ -22,8 +22,8 @@
 ## artifact_modes: types of artifacts to consider in the orientation bias filter (optional)
 ## m2_extra_args, m2_extra_filtering_args: additional arguments for Mutect2 calling and filtering (optional)
 ## split_intervals_extra_args: additional arguments for splitting intervals before scattering (optional)
-## run_orientation_bias_filter: (deprecated) if true, run the orientation bias filter (optional)
-## run_orientation_bias_mixture_model_filter: if true, filter orientation bias sites based on the posterior probabilities computed by the read orientation artifact mixture model (optional)
+## run_orientation_bias_filter: if true, run the orientation bias filter (optional)
+## run_orientation_bias_mixture_model_filter: if true, filter orientation bias sites with the read orientation artifact mixture model, which is the recommended orientation bias filter (optional)
```
[the remark this answers is not in the document]
Comment (takutosato) [reply scripts/mutect2_wdl/mutect2.wdl]: All done. The evaluation results are in the palantir repo: https://github.com/broadinstitute/palantir/pull/671

File: scripts/mutect2_wdl/mutect2.wdl (hunk 1 above)
Comment (takutosato) [reply scripts/mutect2_wdl/mutect2.wdl]: oops

File: scripts/mutect2_wdl/mutect2.wdl (hunk 2 above)
Comment (takutosato) [reply scripts/mutect2_wdl/mutect2.wdl]: done

**Comment by takutosato** [comment]
@LeeTL1220 back to you

**Comment by LeeTL1220** [comment]
@takutosato back to you.

**Comment by LeeTL1220** [comment]
@davidbenjamin @takutosato  I approved this and it needs to be in the next release.  What is holding up a merge?

**Pull request merged**